### DISCUSSION TITLE
Is there a way to get source of a function in CIDER?

### DISCUSSION THREAD

**Discussion by joshcho** [clojure-emacs/cider#3290] [Q&A] [answered]
As in title. In Rebel readline, you can get the source of a function in the REPL. Is that possible in CIDER?

**Comment by vemv** [comment, marked as answer]
Yes. In the spirit of teaching a man how to fish, I'd suggest going over https://docs.cider.mx/cider/index.html and using the search bar.

It also is a good investment of time to simply browse its main sections!
